bundles/org.eclipse.jface/src/org/eclipse/jface/layout/RowLayoutFactory.java

```diff
@@ -336,13 +336,12 @@ public static RowLayout copyLayout(RowLayout layout) {
 		result.marginWidth = layout.marginWidth;
 
 		result.fill = layout.fill;
+		result.center = layout.center;
 		result.justify = layout.justify;
 		result.pack = layout.pack;
 		result.spacing = layout.spacing;
 		result.wrap = layout.wrap;
 
-		result.type = layout.type;
```

Comment:
Why do not copy the type anymore?

Reply:
As mentioned in commit message: The has been already provided via constructor (line 330).

`RowLayout result = new RowLayout(layout.type);`

Reply:
LGTM